Chapters/3-1-Chapter15.md

```diff
@@ -0,0 +1,241 @@
+## Chapter 15
+
+<!--
+- Radio, Voices
+  - Melna confused
+  - Passes listener to Melna
+  - Melna hears static
+    - Lin upset
+    - Listeners only work on Ulukoma
+      - "Oh shit. What do they do now?" moment
+
+- Get sleep.
+-->
+
+"Melna, I think you should see something."
+
+Melna stopped her work abruptly and left the incandescent lamp breathing light onto her makeshift desk to approach a man sitting in the dark. His eyes were fixed on a smoothly polished plate before him while his hands gently fumbled with the damper controls of a large telescopic projector pointed toward the horizon. She appeared behind him, unseen but heard, and he made room for her without looking away.
+
+"Okay Jathi, lead the way," Melna asked.
+
+"Look for a line of small dots rising up from the horizon."
+
+Melna stared into the image plate, searching for any sign of a light in the darkness.
+
+"It's just the sky," she said.
+
+"Close your eyes for a moment, and then open them slowly."
+
+She did as the observer instructed, closing her eyes for longer than she had the patience for, then opened them again. A tiny string of five dots as bright as Ara punctuated the thin yellow-orange of the horizon sky. Jathi swapped out lenses to the highest magnification he had and adjusted the dials ever so delicately to find the dots again. They were still faint, being so close to the horizon, but they had some speed to them and they were rising quickly.
+
+"What are they?" she asked with new urgency.
+
+"Spacecraft. It looks like Holrag has launched something behind Liffa."
+
+Flushed with anger, she rose from the plate and closed her eyes, facing the darkest corner of the room.
+
+"How large are they? Where are they going?" She asked.
+
+The observer answered her, "I can't quite tell their size at this distance, but judging by the speed of the vehicles, I think they're on a similar trajectory as Liffa. I'll have more concrete numbers for you tomorrow."
+
+Melna remained silent.
+
+"What should we do, žah-Melna?,"
+
+Melna took a few deep breaths to regain her composure.
+
+"Holragi wouldn't need any more missions than the one Liffa is on unless something has gone terribly wrong."
+
+"But we've been getting recordings of Liffa's voice every tova'la," said Jathi.
+
+"If they aren't missions, then they must be weapons of some kind. They aren't waiting for Liffa to return."
+
+"Jathi," she turned and approached him. "I want you to record every dot, every time, and find a way to determine their size and trajectory. Pomri will be here tomorrow morning to get your updates before you switch shifts. If anything changes after that, wire your findings to me on the private senate channel. I'm calling for a debate in the evening."
```

Comment:
> I'm calling for a debate in the evening.

srsly? they are going to debate about this? I don't know if the fate of the planet should depend on whoever is the best rhetoric. I know that's how it works here and now... but it's just.... *sigh*